roles/chrony/README.md

```diff
@@ -1,38 +1,47 @@
-Role Name
+# Chrony
 =========
```

Comment:
Now you added new formatting syntax but did not delete the old formatting syntax.

Reply:
The changes was add on the last commit.